Marlin/pins_ANET_10.h

```diff
@@ -24,19 +24,17 @@
  * Anet board pin assignments
  */
 
+#define BOARD_NAME "Anet"
+
 #if defined(__AVR_ATmega1284P__)
 #define VARIANT_STANDARD true
 #endif
 
 #include "pins_SANGUINOLOLU_12.h"
 
-#undef BOARD_NAME
-#define BOARD_NAME "Anet"
-
 #undef LED_PIN
 #undef FAN_PIN
 
-#define LED_PIN -1
 #define FAN_PIN 4
```

Comment:
thinkyhead was reviewing my code and told it. So just create a pull request, and you will see what the say.